app/views/questions/show.erb

```diff
@@ -1,3 +1,8 @@
+
 <div class="container">
-  here are specific question details
-</div>
\ No newline at end of file
+
+  <p><h4><%= @question.title %></h4> <h6><%= User.find_by_id(@question.user_id).username%></p>
```

Comment:
don't forget that this kind of logic should be reserved for the route: <%= User.find_by_id(@question.user_id).username%>. If our question show route has already found the question in mind, then our view should only display "<%=@question.user.username%>" according to our associations.